ci-operator/step-registry/telcov10n/sno-hub-ztp/share-hub-cluster-profile/telcov10n-sno-hub-ztp-share-hub-cluster-profile-commands.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ Fix container user ************"
+# Fix user IDs in a container
+[ -e "${HOME}/fix_uid.sh" ] && "${HOME}/fix_uid.sh" || echo "${HOME}/fix_uid.sh was not found" >&2
+
+function setup_aux_host_ssh_access {
+
+  echo "************ telcov10n Setup AUX_HOST SSH access ************"
+
+  SSHOPTS=(
+    -o 'ConnectTimeout=5'
+    -o 'StrictHostKeyChecking=no'
+    -o 'UserKnownHostsFile=/dev/null'
+    -o 'ServerAliveInterval=90'
+    -o LogLevel=ERROR
+    -i "${CLUSTER_PROFILE_DIR}/ssh-key"
+  )
+
+}
+
+function save_hub_cluster_profile_artifacts {
+
+  echo "************ telcov10n Save those artifacts that will be used during Spoke deployments ************"
+
+  hub_to_spoke_artifacts=(
+    "$(readlink -f ${KUBEADMIN_PASSWORD_FILE})"
+    "$(readlink -f ${CLUSTER_PROFILE_DIR}/pull-secret)"
+    "$(readlink -f ${CLUSTER_PROFILE_DIR}/base_domain)"
+  )
+
+  local_cluster_profile_shared_folder=$(mktemp -d --dry-run)/${SHARED_HUB_CLUSTER_PROFILE}
+  mkdir -pv ${local_cluster_profile_shared_folder}
+  cp -av "${hub_to_spoke_artifacts[@]}" ${local_cluster_profile_shared_folder}/
+  cp -v  "$(readlink -f ${KUBECONFIG})" ${local_cluster_profile_shared_folder}/hub-kubeconfig
+
+  echo
+  set -x
+  rsync -avP --delete-before \
+    -e "ssh $(echo "${SSHOPTS[@]}")" \
+    "${local_cluster_profile_shared_folder}" \
+    "root@${AUX_HOST}":/var/builds/${NAMESPACE}
+  set +x
+  echo
+}
+
+function create_symbolic_link_to_shared_hub_cluster_profile_artifacts_folder {
+  
+  echo "************ telcov10n Create a symbolic link to the shared artifacts folder that would be used during Spoke deployments ************"
+
+  ln -s /var/builds/${NAMESPACE}/${SHARED_HUB_CLUSTER_PROFILE} ${SHARED_HUB_CLUSTER_PROFILE}
+
+  echo
+  set -x
+  rsync -avP \
+    -e "ssh $(echo "${SSHOPTS[@]}")" \
+    "${SHARED_HUB_CLUSTER_PROFILE}" \
+    "root@${AUX_HOST}":/var/telco-qe-preserved/
```

Comment:
```suggestion
    "root@${AUX_HOST}":/var/builds/telco-qe-preserved/
```

@sgoveas do we iterate through the foders in /var/builds?

Reply:
Yes
https://github.com/openshift-qe/baremetal-qe-infra/blob/master/images/fcos-bastion-image/root/usr/bin/clean-up.sh#L16